tests/lcs/core/graphics/graphics.livecodescript

```diff
@@ -90,4 +90,120 @@ TestAssert "test", not ("4,9" is not within "-5,-10,5,10")
 TestAssert "test", "-6,-11" is not within "-5,-10,5,10"
 TestAssert "test", "5,10" is not within "-5,-10,5,10"
 
-end TestGraphicsIsNotWithin
\ No newline at end of file
+end TestGraphicsIsNotWithin
+
+-- new brush mapping for 101..135
+-- brush < 101 = brush
+-- 101 < brush < 135 = brush - 100
+-- brush > 136 = brush
+
+on TestBrushWithNoIcons
+	TestAssert "set brush with revicons stack not loaded", SetBrushNoIcons() is not 2
+end TestBrushWithNoIcons
+
+on TestBadBrush100
+	open stack "../ide/Toolset/palettes/revicons.rev"
+	set the brush to 8
+	TestAssert "no brush with id=100", SetBrush100() is not 100
+	TestAssert "brush image not found", the brush is 8
+end TestBadBrush100
+
+on TestBadBrushZero
+	open stack "../ide/Toolset/palettes/revicons.rev"
+	set the brush to 8
+	TestAssert "no brush with id=0", SetBrushZero() is not zero
+	TestAssert "brush image not found", the brush is 8
+end TestBadBrushZero
+
+function SetBrushNoIcons
+	try
+		set the brush to 2
+	catch e
+		TestDiagnostic e
+	end try
+	return the brush
+end SetBrushNoIcons
+
+function SetBrush100
+	try
+		set the brush to 100
+	catch e
+		TestDiagnostic e
+	end try
+	return the brush
+end SetBrush100
+
+function SetBrushZero
+	try
+		set the brush to 0
+	catch e
+		TestDiagnostic e
+	end try
+	return the brush
+end SetBrushZero
+
+on TestBrush
+	open stack "../ide/Toolset/palettes/revicons.rev"
+	set the brush to 30
+	TestAssert "brush > 35 unchanged", the brush is 30
+end TestBrush
+
+on TestRemappedBrush
+	local tBrush
+	
+	open stack "../ide/Toolset/palettes/revicons.rev"
+#	repeat with tBrush = 101 to 102
```

Comment:
@mwieder we should write the tests based on how we know the engine should perform and work out why they are failing if they fail. Perhaps the first test to run once we load revicons is on the existence of image id 101-136?